kadi/settings.py

```diff
@@ -87,35 +87,24 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-TEMPLATE_DEBUG = True
-
 ALLOWED_HOSTS = []
 
 
 # Application definition
 
 INSTALLED_APPS = (
     'django.contrib.admin',
-    # 'django.contrib.admindocs',  # ?? needed?  Lost in 1.6
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'kadi.events',
+    'mica.web',
+    'find_attitude.web_find_attitude',  # app label (last module) must be unique
 )
 
-OPTIONAL_APPS = ('mica.web', 'find_attitude.web')
```

Comment:
It wasn't clear to me from the PR notes that mica.web and find_attitude.web are now not optional (from this code cut, I thought maybe their optional nature was just handled via a different import mechanism).  I don't really care in this case, just trying to keep an eye on dependencies

Reply:
The OPTIONAL_APPS was not working with django 3 and so I decided to not spend time on trying to fix that and instead just made them required.

Reply:
But in fairness these are both listed in the **Skare3 package requirements** up top.

Reply:
Yes, I saw that.  It just wasn't clear to me that the find_attitude and mica were required to run kadi's built in tests (because the tests below included the web server stuff, and it was obvious they would be needed there).  For me, this just changes what I want in the kadi conda recipe, but I know you don't care about those conda dependencies anyway.